pkg/workflows/create.go

```diff
@@ -359,9 +371,49 @@ func (s *DeleteBootstrapClusterTask) Run(ctx context.Context, commandContext *ta
 	if commandContext.OriginalError == nil {
 		logger.MarkSuccess("Cluster created!")
 	}
-	return nil
+	return &InstallPackageControllerTask{}
 }
 
 func (s *DeleteBootstrapClusterTask) Name() string {
 	return "delete-kind-cluster"
 }
+
+func (s *InstallPackageControllerTask) Run(ctx context.Context, commandContext *task.CommandContext) task.Task {
+	logger.Info("Installing curated packages controller on workload  cluster")
+	kubeConfig := kubeconfig.FromClusterName(commandContext.ClusterSpec.Cluster.Name)
+	deps, err := curatedpackages.NewDependenciesForPackages(ctx, kubeConfig)
+	if err != nil {
+		logger.MarkFail("Error when installing curated packages on workload cluster; please install through eksctl anywhere install command", "error", err)
```

Comment:
Same as the other comment on logging. Seems like this is called during `install packagecontroller` cmd so it seems unhelpful. Can you clarify?

Reply:
Yes, this is to account for failure of curated packages controller during cluster creation. There is a separate command you could use to run this which is namely the install. I'll add better details.

Reply:
Thanks.